### DISCUSSION TITLE
rs服务器返回200，apisix返回499

### DISCUSSION THREAD

**Discussion by Caibreeze0908** [apache/apisix#6058] [Q&A]
apisix2.4
Static resource (HTML, PNG, CSS) files are requested using HTTP.2.0. The back-end real server returns 200 and a small portion of apisix returns 499 with a request time and response time of less than 1s

**Comment by tokers** [comment]
499 means the downstream connections (client) are aborted before APISIX can send any bytes. It's tough to say the reason if we don't know the client behaviors and the network quality between your clients and APISIX.

**Reply by Caibreeze0908** [reply]
After testing the client and apisix network is no problem, and only request HTML, CSS files, the file size is very small
